tools/distrib/python/make_grpcio_tools.py

```diff
@@ -125,8 +127,8 @@ def get_deps():
     ]
     commit_hash = protobuf_submodule_commit_hash()
     deps_file_content = DEPS_FILE_CONTENT.format(
-        cc_files=cc_files,
-        proto_files=proto_files,
+        cc_files=json.dumps(sorted(cc_files), indent=4),
```

Comment:
Nit: [`pprint`](https://docs.python.org/3/library/pprint.html) is guaranteed to produce valid Python. This should be okay for lists, but if there's a bug before this, the error could be a bit confusing.

Reply:
Thanks for the tip. Done (but needed an adjustment to end up with the format I wanted).